audinterface/core/process.py

```diff
@@ -819,7 +820,7 @@ def process_signal_from_index(
         y = self(signal, sampling_rate, starts_i, ends_i)
 
         # For an empty Series we force the dtype
-        if len(y) == 0:
+        if isinstance(y, collections.abc.Iterable) and len(y) == 0:
```

Comment:
I guess we added the check for an empty sequence for the special case that the index is empty. But of course check this before we actually do the processing, then we can remove the `if-else` case here.